src/modules/auth/resources/ResourcesListContainer.js

```diff
@@ -65,21 +89,53 @@ function ResourcesListContainer(props) {
 		} catch(e) {
 			console.error(e);
 			setLoading(false);
-			props.app.addAlert("warning", `${t("ResourcesListContainer|Something went wrong, failed to load resources")}. ${e?.response?.data?.message}`, 30);
+			props.app.addAlert("warning", `${t("ResourcesListContainer|Failed to load resources")}. ${e?.response?.data?.message}`, 30);
+		}
+	}
+	// Set terminate resource dialog
+	const terminateResourceForm = (resourceId) => {
+		var r = confirm(t('ResourcesListContainer|Do you want to delete this resource'));
+		if (r == true) {
+			deleteResource(resourceId);
 		}
 	}
 
-	const createResourceComponent = (
-		<ButtonWithAuthz
-			title={t("ResourcesListContainer|Create resource")}
-			color="primary"
-			onClick={() => {redirectToCreate()}}
-			resource="authz:superuser"
-			resources={credentialsResources}
-		>
-			{t("ResourcesListContainer|Create resource")}
-		</ButtonWithAuthz>
-	)
+	// Terminate the resource
+	const deleteResource = async (resourceId) => {
```

Comment:
@petrKavulok in some files, you specify `resource_id` prop as a `res` and here you have `resourceId`, so please unify across the files (rather with `resourceId` which is more descriptive than `res`)